pwa/src/pages/entities/index.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from "react";
-import Layout from "../../components/common/layout";
 import EntitiesTable from "../../components/entities/entitiesTable";
 
 const IndexPage = () => {
   return (
-    <Layout title={"Object"} subtext={"An overview of your objects"}>
+    <>
       <title>Gateway - Objects</title>
```

Comment:
What does this actually do? I don't see it anywhere.

Also, this is not included in the other similar files. We might be able to just delete it and then also delete the empty `<></>` wrapper because `<main>` would wrap around everything.